pkg/controller/operators/catalog/operator.go

```diff
@@ -416,6 +501,27 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 	jobInformer := k8sInformerFactory.Batch().V1().Jobs()
 	sharedIndexInformers = append(sharedIndexInformers, jobInformer.Informer())
 
+	if err := labelObjects(batchv1.SchemeGroupVersion.WithResource("jobs"), jobInformer.Informer(), labeller.ObjectLabeler[*batchv1.Job, *batchv1applyconfigurations.JobApplyConfiguration](
+		ctx, op.logger, func(object metav1.Object) bool {
+			job, ok := object.(*batchv1.Job)
+			if !ok {
+				return false
+			}
+			for _, container := range job.Spec.Template.Spec.Containers {
+				if strings.Join(container.Command[0:3], " ") == "opm alpha bundle extract" {
+					return true
```

Comment:
If something/someone else runs a job that looks like this, what are the implications? We'll apply a label to it and then start getting events/copies in our cache?

Does (or, in the future, could) that label imply OLM ownership and therefore other side-effects that shouldn't happen with what is intended to be a non-OLM-owned job?